doc/api/util.md

```diff
@@ -125,6 +134,9 @@ util.format(1, 2, 3); // '1 2 3'
 ```
 
 ## util.inherits(constructor, superConstructor)
+<!-- YAML
+added: v0.3.0
```

Comment:
This is when the module has been renamed. It was called `sys` before, not sure if it's better to show when the method was added to `sys`.